src/_guides/packages.md

```diff
@@ -54,6 +54,18 @@ dependencies:
   intl: ^0.17.0
 ```
 
+Rather than editing the pubspec manually, 
+you can also add a dependency by running a `dart` tool command. 
+This adds a dependency on `vector_math`: 
```

Comment:
**issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To update the `pubspec.yaml` file, you have a better option than hand editing. 
To add a dependency, you can run `dart pub add` command.
The following example adds a dependency on `vector_math`.
```

Reply:
Thanks! Committed a variant of that (wanted to avoid "hand" editing)